src/code/audio_synthesis.c

```diff
@@ -459,12 +458,12 @@ void func_800DBE64(void) {
 void func_800DBE6C(void) {
 }
 
-void AudioSynth_LoadFilter(Acmd* cmd, s32 flags, s32 countOrBuf, void* addr) {
-    aFilter(cmd, flags, countOrBuf, addr);
+void AudioSynth_LoadFilterBuffer(Acmd* cmd, s32 flags, s32 buf, void* addr) {
+    aFilter(cmd, flags, buf, addr);
 }
 
-void AudioSynth_LoadFilterCount(Acmd* cmd, s32 count, void* addr) {
-    aFilter(cmd, 2, count, addr);
+void AudioSynth_LoadFilterSize(Acmd* cmd, s32 size, void* addr) {
+    aFilter(cmd, 2, size, addr);
```

Comment:
Looking at the ucode, I think with flags=2 the `countOrBuf` argument is ignored... but I'm not confident

Reply:
`flags = 2` is an indication to read the `sizeOrBuf` as a size